### PR TITLE
BBE Landing: Update FAQ copy

### PR DESCRIPTION
**Pull request by aneeshd16**
Fixes Automattic/martech#1569

## Proposed Changes

* Updates the FAQ text on the BBE landing page. Please check the linked issue for more details.
* **Note: [String Freeze] label to be added after review.**

## Testing Instructions

* Go to `/setup/site-setup/difmStartingPoint?siteSlug=<site slug>`, and confirm that the copy matches the one in the linked issue.

calypso localhost_3000_setup_site-setup_difmStartingPoint_siteSlug=adadsadsaorg wordpress com

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by jeyip** [comment]
## Testing
### Requirements
- [x] Confirm that the copy matches the new Built By Express - Calypso Landing page FAQ copy determined by Mike and Aneesh

### Browsers
- [x] Chrome

**Notes:**
Rerunning e2e specs

File: client/my-sites/marketing/do-it-for-me/difm-landing.tsx [hunk 1]
```diff
@@ -238,16 +232,24 @@ export default function DIFMLanding( {
 	const product = useSelector( ( state ) => getProductBySlug( state, WPCOM_DIFM_LITE ) );
 	const productCost = product?.cost;
 
+	const planObject = getPlan( PLAN_PREMIUM );
+	const planTitle = planObject?.getTitle();
```
Comment (jeyip) [inline client/my-sites/marketing/do-it-for-me/difm-landing.tsx:236]: `Non-blocking / Nit:`

I didn't see this explicitly state in the P2 discussion, but it looks like we're no longer considering the `PLAN_WPCOM_PRO` when generating FAQ text. Just wanted to confirm this is intentional 🙂

**Review by jeyip — APPROVED** [review]
🚢

File: client/my-sites/marketing/do-it-for-me/difm-landing.tsx (hunk 1 above)
Comment (aneeshd16) [reply client/my-sites/marketing/do-it-for-me/difm-landing.tsx:236]: Thanks for the review, Jeremy! Yes, the Pro plan was removed long ago, but somehow this check was still in the code base.

**Comment by aneeshd16** [comment]
Hi @mikeicode, can you please review if the copy changes are okay?

**Comment by mikeicode** [comment]
> Hi @mikeicode, can you please review if the copy changes are okay?

@aneeshd16 I reviewed, changes look great!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8045682

Thank you @aneeshd16 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8045682).

**Pull request merged**